packages/subgraph/tests/addresslist-voting/addresslist-voting.test.ts

```diff
@@ -53,13 +48,13 @@ import {
   createAddresslistVotingProposalEntityState,
 } from './utils';
 import {
-  generateEntityIdFromAddress,
   generatePluginEntityId,
+  createDummyAction,
 } from '@aragon/osx-commons-subgraph';
-import {Address, BigInt, Bytes, log} from '@graphprotocol/graph-ts';
+import {Address, BigInt} from '@graphprotocol/graph-ts';
 import {assert, clearStore, test} from 'matchstick-as/assembly/index';
 
-let actions = createDummyActions(DAO_TOKEN_ADDRESS, '0', '0x00000000');
+let action = createDummyAction(DAO_TOKEN_ADDRESS, '0', '0x00000000');
```

Comment:
```suggestion
let actions = [createDummyAction(DAO_TOKEN_ADDRESS, '0', '0x00000000')];
```

Why don't we create the array here so we don't need to create array [action] in the rest of the test?